DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzContentOnTheFlyBundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+class Configuration implements ConfigurationInterface
+{
+    public function getConfigTreeBuilder()
+    {
+        $treeBuilder = new TreeBuilder();
+        $rootNode = $treeBuilder->root('ez_systems_ez_content_on_the_fly');
```

Comment:
I would suggest to change it to `ez_content_on_the_fly` or `ez_contentonthefly` to be consistent with other bundles.